app/models/manageiq/providers/vmware/infra_manager/inventory/parser/virtual_machine.rb

```diff
@@ -296,6 +296,7 @@ def parse_virtual_machine_snapshot(vm, snapshot, current, parent_uid = nil)
       return if snap.nil?
 
       create_time = snapshot[:createTime]
+      parent = persister.snapshots.lazy_find(:vm_or_template => vm, :uid => Time.parse(parent_uid).iso8601(6)) if parent_uid
```

Comment:
hm, it's strange that the uid here is different to uid below. Those should be the same, was there some problem with it?

Reply:
ah nevermind, you parse the parent_uid, not the create_time, ok :+1: